### DISCUSSION TITLE
Budget and Category.

### DISCUSSION THREAD

**Discussion by M1scer** [firefly-iii/firefly-iii#7929] [Firefly III]
Hi,

according to the docs thats how i should handle budgets and categorys:```

```
If` you try to save money every month on a certain subject, it's a budget. Groceries are budget. Bills are a budget. If you travel by train occasionally, it's not a budget.

A category is "incidental". You don't buy new furniture every month, but you might want to keep track of such expenses. Or you don't care about costs for public traffic (budget-wise) but a category would be nice.

The rule of thumb is: would you make a real life envelope for it? If yes: budget. If no: category.
```

Now i recognized, that all my "budget transactions" are listed in the "no category" section.
Shouldn't they be ignored? The idea is to have only budget or category?!

A other solution would be to add a category to all my "budget transactions".

What is the best way to handle this?

**Comment by JC5** [comment]
You can give transactions both a budget and a category. Nothing wrong with that.
